app/components/events/view/overview/event-apps.js

```diff
@@ -1,7 +1,17 @@
 import classic from 'ember-classic-decorator';
 import { classNames } from '@ember-decorators/component';
 import Component from '@ember/component';
+import { computed } from '@ember/object';
+import ENV from 'open-event-frontend/config/environment';
 
 @classic
 @classNames('ui', 'fluid', 'card')
-export default class EventApps extends Component {}
+export default class EventApps extends Component {
+    @computed('eventId')
+  get webAppGeneratorUrl() {
```

Comment:
Indention error

Reply:
@iamareebjamal I think it is fine because it got automatically indented like this when I did git commit. Let me know how many spaces need to be included.

Reply:
2